### DISCUSSION TITLE
How to hide language "Jupyter Notebook" in top-langs stats?

### DISCUSSION THREAD

**Discussion by sohang3112** [anuraghazra/github-readme-stats#3568] [Q&A] [answered]
I tried to hide Jupyter Notebooks from showing on my *top-langs* stats using [hide parameter](https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#hide-individual-languages), but nothing happened:

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sohang3112&hide=SCSS&layout=compact&theme=gruvbox_light?hide=jupyternotebook)](https://github.com/anuraghazra/github-readme-stats)
```

I tried these in `hide` value, but none of them worked: `jupyternotebook`, `jupyter-notebook`, `Jupyter%20Notebook`, `ipynb`.

**Comment by FajarKim** [comment, marked as answer]
@sohang3112 There is an error in the link you provided. This should work:

```markdown
![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sohang3112&hide=scss,jupyter%20notebook&layout=compact&theme=gruvbox_light)
```

Result:

![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sohang3112&hide=scss,jupyter%20notebook&layout=compact&theme=gruvbox_light)

**Reply by rickstaa** [reply]
@FajarKim Thanks for answering this question. @sohang3112, @FajarKim is right; we expect the language hide input to use [percent encoding](https://en.wikipedia.org/wiki/Percent-encoding) (see https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#language-card-exclusive-options).

**Reply by sohang3112** [reply]
Thanks @FajarKim @rickstaa - this worked 🙂
